node/pegnet/grading.go

```diff
@@ -0,0 +1,65 @@
+package pegnet
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pegnet/pegnet/modules/opr"
+)
+
+const createTableGrade = `CREATE TABLE IF NOT EXISTS "pn_grade" (
+	"height" INTEGER NOT NULL,
+	"winners" BLOB,
+	
+	UNIQUE("height")
+);
```

Comment:
I think it'd be nice to have some more context around the eblock keymr and sequence.

If we only index by height, we can only traverse by height, and there will be gaps. If we also store
```
Eblock keymr
Eblock Seq
Prev KeyMr
```

We can traverse chains cleanly. I could see value in this indexing. I think it would also be a nice index to store the previous keymr/height/seq (choose one, idc) of the last eblock with a valid graded block. Since not all eblocks necesarily have a set of winners.

Reply:
traversing by height is the same thing as traversing by prevkeymr and sequence but way better than prevkey. sequence has some merits but we're not really doing anything by sequence in here. storing the previous index is pointless in an sql environment, you only do that in key-value databases


i think we should store the eblock keymr for lookup purposes but i don't think the others are necessary

Reply:
i ended up just adding them since it didn't really cost anything